### PR TITLE
GovernorBravo UI 

### PR DESCRIPTION
**Pull request by RoseRompkxm**
Here comes the UI changes adjusted to our brand new governor implementation - Governor Bravo
- a new vote type `abstain` is supported
- votes can now vote with reasons
- UI refined

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN apk add --update --no-cache python3 git openssh
 
 WORKDIR /usr/app
 
-COPY package.json package-lock.json ./
+COPY package.json ./
```
Comment (kkirka) [inline Dockerfile]: Locking the deps is generally a good thing to do, do you think we should revert to unlocked deps?

File: src/components/Basic/Proposal.js [hunk 2]
```diff
@@ -164,17 +256,29 @@ function Proposal({ address, proposal, votingWeight, history }) {
...
-  const handleVote = async support => {
+  const handleOpenVoteConfirmModal = type => {
+    setVoteType(type);
+    setConfirmModalVisible(true);
+  };
+
+  const handleVote = async () => {
     setIsLoading(true);
-    setVoteType(support);
+    console.log(voteType);
```
Comment (kkirka) [inline src/components/Basic/Proposal.js]:
```suggestion
```

File: src/containers/Main/VoteOverview.js [hunk 3]
```diff
@@ -257,76 +271,42 @@ function VoteOverview({ settings, getVoters, getProposalById, match }) {
...
+                      label={data.label}
+                      voteNumber={new BigNumber(data.votes.sumVotes)}
+                      totalNumber={totalVotes}
+                      addressNumber={
+                        new BigNumber(data.votes.total).isNaN()
+                          ? 0
+                          : data.votes.total
+                      }
+                      emptyNumber={
+                        4 -
```
Comment (kkirka) [inline src/containers/Main/VoteOverview.js]: 🤔 maybe

`const displayedRows = 4;`
`emptyNumber={ displayedRows - ... }`

File: src/hooks/useContract.js [hunk 4]
```diff
@@ -81,5 +82,11 @@ export const useXvsVault = () => {
 
 export const useXvsVaultProxy = () => {
   const web3 = useWeb3();
-  return useMemo(() => getXvsVaultProxyContract(web3), [web3]);
+  // todo
```
Comment (kkirka) [inline src/hooks/useContract.js]: Todo what? 🤔

**Review by kkirka — APPROVED** [review]

File: src/hooks/useContract.js (hunk 4 above)
Comment (RoseRompkxm) [reply src/hooks/useContract.js]: some undeleted comments while deving.. 🤣

File: src/components/Basic/Proposal.js (hunk 2 above)
Comment (RoseRompkxm) [reply src/components/Basic/Proposal.js]: got it

File: Dockerfile (hunk 1 above)
Comment (RoseRompkxm) [reply Dockerfile]: no, let's keep lock file

**Pull request merged**